UserSettings/CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.2.0] - 2024-09-17
```

Comment:
Need to also change https://github.com/rdkcentral/rdkservices/blob/main/UserSettings/UserSettings.cpp#L23 to update the version